v3/src/data-interactive/data-interactive-types.ts

```diff
@@ -94,14 +95,14 @@ export interface DIResources {
 // types for values accepted as inputs by the API
 export type DISingleValues = DIAttribute | DINotifyAttribute | DIAttributeLocationValues | DICase | DIDataContext |
   DINotifyDataContext | DIGlobal | DIInteractiveFrame | DIItemValues | DICreateCollection | DINewCase | DIUpdateCase |
-  DINotification | DIItemSearchNotify | DILogMessage | DIUrl | V2SpecificComponent
+  DINotification | DIItemSearchNotify | DILogMessage | DIUrl | V2SpecificComponent | DICategoryColorMap
 export type DIValues = DISingleValues | DISingleValues[] | number | string[]
 
 // types returned as outputs by the API
 export type DIResultSingleValues = DICase | DIComponentInfo |  DIDataDisplay | DIGetCaseResult | DIGlobal
   | DIInteractiveFrame
 
-export type DIResultValues = DIResultSingleValues | DIResultSingleValues[] |
+export type DIResultValues = DIResultSingleValues | DIResultSingleValues[] | DICategoryColorMap |
```

Comment:
```suggestion
export type DIResultValues = DIResultSingleValues | DIResultSingleValues[] |
```
Shouldn't be necessary now that colormap isn't a separate API call.